pkg/controller/backup_sync_controller_test.go

```diff
@@ -17,18 +17,20 @@ limitations under the License.
 package controller
 
 import (
-	"errors"
 	"testing"
 	"time"
 
-	"github.com/stretchr/testify/assert"
-
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/sets"
 	core "k8s.io/client-go/testing"
 
 	"github.com/heptio/ark/pkg/apis/ark/v1"
 	"github.com/heptio/ark/pkg/generated/clientset/versioned/fake"
 	"github.com/heptio/ark/pkg/util/stringslice"
 	arktest "github.com/heptio/ark/pkg/util/test"
+	"github.com/pkg/errors"
```

Comment:
Were these moved by an imports tool?

Reply:
My previously-suggested stance on imports is there are more important things to worry about in the review than import grouping and order. In this case, it was a switch from `errors` to `github.com/pkg/errors` and not a straight move. I'm not sure why `testify/assert` moved. But overall my suggestion is to leave it alone. WDYT?

Reply:
Good point - I was mostly curious here